Plugins~/Src/MeshSyncClientBlender/msblenBinder.h

```diff
@@ -6,6 +6,19 @@
 
 struct Depsgraph;
 
+// The data structures have changed in blender 3.5
+// These data structures are wrappers for the new data with the fields we need to keep the plugin code for different blender versions the same:
```

Comment:
I have some trouble understanding this comment, do you mean that these structures act as an interface for different version of blender?

Reply:
Yeah since 3.5 these data structures do not exist anymore so we make our own and put the data into them to match the format from previous versions so the external interface outside msblenBinder remains the same. Without this we'd have lots of custom code for blender 3.5 everywhere.